packages/busybox.rb

```diff
@@ -14,1200 +14,1205 @@ class Busybox < Package
-      CONFIG_FEATURE_WGET_TIMEOUT=y
-      CONFIG_FEATURE_WGET_HTTPS=y
-      CONFIG_FEATURE_WGET_OPENSSL=y
-      CONFIG_WHOIS=y
-      CONFIG_ZCIP=y
-      # CONFIG_UDHCPD is not set
-      # CONFIG_FEATURE_UDHCPD_BASE_IP_ON_MAC is not set
-      # CONFIG_FEATURE_UDHCPD_WRITE_LEASES_EARLY is not set
-      CONFIG_DHCPD_LEASES_FILE=""
-      # CONFIG_DUMPLEASES is not set
-      # CONFIG_DHCPRELAY is not set
-      # CONFIG_UDHCPC is not set
-      # CONFIG_FEATURE_UDHCPC_ARPING is not set
-      # CONFIG_FEATURE_UDHCPC_SANITIZEOPT is not set
-      CONFIG_UDHCPC_DEFAULT_SCRIPT=""
-      # CONFIG_UDHCPC6 is not set
-      # CONFIG_FEATURE_UDHCPC6_RFC3646 is not set
-      # CONFIG_FEATURE_UDHCPC6_RFC4704 is not set
-      # CONFIG_FEATURE_UDHCPC6_RFC4833 is not set
-      # CONFIG_FEATURE_UDHCPC6_RFC5970 is not set
-      # CONFIG_FEATURE_UDHCP_PORT is not set
-      CONFIG_UDHCP_DEBUG=0
-      CONFIG_UDHCPC_SLACK_FOR_BUGGY_SERVERS=0
-      # CONFIG_FEATURE_UDHCP_RFC3397 is not set
-      # CONFIG_FEATURE_UDHCP_8021Q is not set
-      CONFIG_IFUPDOWN_UDHCPC_CMD_OPTIONS="-R -n"
+    #
+    # Networking Utilities
+    #
+    CONFIG_FEATURE_IPV6=y
+    # CONFIG_FEATURE_UNIX_LOCAL is not set
+    CONFIG_FEATURE_PREFER_IPV4_ADDRESS=y
+    CONFIG_VERBOSE_RESOLUTION_ERRORS=y
+    # CONFIG_FEATURE_TLS_SHA1 is not set
+    # CONFIG_ARP is not set
+    CONFIG_ARPING=y
+    # CONFIG_BRCTL is not set
+    # CONFIG_FEATURE_BRCTL_FANCY is not set
+    # CONFIG_FEATURE_BRCTL_SHOW is not set
+    # CONFIG_DNSD is not set
+    CONFIG_ETHER_WAKE=y
+    CONFIG_FTPD=y
+    CONFIG_FEATURE_FTPD_WRITE=y
+    CONFIG_FEATURE_FTPD_ACCEPT_BROKEN_LIST=y
+    CONFIG_FEATURE_FTPD_AUTHENTICATION=y
+    CONFIG_FTPGET=y
+    CONFIG_FTPPUT=y
+    CONFIG_FEATURE_FTPGETPUT_LONG_OPTIONS=y
+    CONFIG_HOSTNAME=y
+    CONFIG_DNSDOMAINNAME=y
+    # CONFIG_HTTPD is not set
+    # CONFIG_FEATURE_HTTPD_RANGES is not set
+    # CONFIG_FEATURE_HTTPD_SETUID is not set
+    # CONFIG_FEATURE_HTTPD_BASIC_AUTH is not set
+    # CONFIG_FEATURE_HTTPD_AUTH_MD5 is not set
+    # CONFIG_FEATURE_HTTPD_CGI is not set
+    # CONFIG_FEATURE_HTTPD_CONFIG_WITH_SCRIPT_INTERPR is not set
+    # CONFIG_FEATURE_HTTPD_SET_REMOTE_PORT_TO_ENV is not set
+    # CONFIG_FEATURE_HTTPD_ENCODE_URL_STR is not set
+    # CONFIG_FEATURE_HTTPD_ERROR_PAGES is not set
+    # CONFIG_FEATURE_HTTPD_PROXY is not set
+    # CONFIG_FEATURE_HTTPD_GZIP is not set
+    # CONFIG_FEATURE_HTTPD_ETAG is not set
+    # CONFIG_FEATURE_HTTPD_LAST_MODIFIED is not set
+    # CONFIG_FEATURE_HTTPD_DATE is not set
+    # CONFIG_FEATURE_HTTPD_ACL_IP is not set
+    CONFIG_IFCONFIG=y
+    CONFIG_FEATURE_IFCONFIG_STATUS=y
+    CONFIG_FEATURE_IFCONFIG_SLIP=y
+    CONFIG_FEATURE_IFCONFIG_MEMSTART_IOADDR_IRQ=y
+    CONFIG_FEATURE_IFCONFIG_HW=y
+    CONFIG_FEATURE_IFCONFIG_BROADCAST_PLUS=y
+    CONFIG_IFENSLAVE=y
+    CONFIG_IFPLUGD=y
+    CONFIG_IFUP=y
+    CONFIG_IFDOWN=y
+    CONFIG_IFUPDOWN_IFSTATE_PATH="/usr/local/var/lib/ifstate"
```

Comment:
This should be abstracted with `#{CREW_PREFIX}` in place of `/usr/local`.

Reply:
yk what, let's just `sed -i "s:/usr/local:#{CREW_PREFIX}:g" busybox.rb`